sim/paladin/item_sets_pve.go

```diff
@@ -8,6 +8,38 @@ import (
 	"github.com/wowsims/sod/sim/core/stats"
 )
 
+// Soul Related Set Bonus IDs
+const (
+	PaladinT1Prot2P  = 456536
+	PaladinT1Prot4P  = 456538
+	PaladinT1Prot6P  = 456541
+	PaladinT2Prot2P  = 467531
+	PaladinT2Prot4P  = 467532
+	PaladinT2Prot6P  = 467536
+	PaladinTAQProt2P = 1213410
+	PaladinTAQProt4P = 1213413
+	PaladinT1Holy2P  = 456488
+	PaladinT1Holy4P  = 457323
+	PaladinT1Holy6P  = 456492
+	PaladinT2Holy2P  = 467506
+	PaladinT2Holy4P  = 467507
+	PaladinT2Holy6P  = 467513
+	PaladinTAQHoly2P = 1213349
+	PaladinTAQHoly4P = 1213353
+	PaladinT1Ret2P   = 456494
+	PaladinT1Ret4P   = 456489
+	PaladinT1Ret6P   = 456533
+	PaladinT2Ret2P   = 467518
+	PaladinT2Ret4P   = 467526
+	PaladinT2Ret6P   = 467529
+	PaladinTAQRet2P  = 1213397
+	PaladinTAQRet4P  = 1213406
+	PaladinZG2P      = 468401
+	PaladinZG3P      = 468428
+	PaladinZG5P      = 468431
+	PaladinRAQ3P     = 123467
+)
```

Comment:
Can these be deleted now?

Reply:
I'm using them for the auras to track duplicates.